styles/dusk-2.json

```diff
@@ -0,0 +1,321 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Dusk",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"color": "#E2E2E2",
```

Comment:
As per figma 

Base color is: `#E5E5E5`,
contrast: `#191919`,
Accent-1: `#650DD4`,
Accent-2: `#FFFFFF`,
Primary: `#3B3B3B`

All these colors are slightly different, and some are assigned to different slugs.

Reply:
The colors are the same as in the dusk color palette. 
The **names** in Figma have not been updated to match the names needed in the theme.

Reply:
Got it. In Figma it's wrong: https://prnt.sc/k9H-Aq6-sqA7

Reply:
I'll add the "needs design feedback" label.